setup.py

```diff
@@ -43,6 +43,7 @@
             version=VERSION,
             packages=PACKAGES,
             package_data=PACKAGE_DATA,
+            zip_safe=False,
```

Comment:
So we're switching from `distutils.core` to `setuptools` and `setuptools` installs the package in the egg binary format. Setting `zip_safe` to `False` bypasses that whole calculation and installs both the egg and source formats in site-packages.

https://pythonhosted.org/setuptools/setuptools.html#setting-the-zip-safe-flag

One alternative would be using the wheel format.